### PR TITLE
Fix double scroll on mobile and related questions filter

### PR DESCRIPTION
**Pull request by Aprillion**

### CODE REVIEW

File: app/hooks/useQuestionStateInUrl.ts [hunk 1]
```diff
@@ -116,6 +123,7 @@ export default function useQuestionStateInUrl(minLogo: boolean, initialQuestions
 
     const onSiteAnswers = onSiteAnswersRef.current
     const onSiteSet = new Set(onSiteAnswers.map(({pageid}) => pageid))
+    console.debug(relatedQuestions, onSiteSet)
```
Comment (mruwnik) [inline app/hooks/useQuestionStateInUrl.ts:126]: should this be here?

File: app/server-utils/stampy.ts [hunk 2]
```diff
@@ -283,13 +283,15 @@ export const loadTag = withCache('tag', async (tagName: string): Promise<Tag> =>
 
   const questions = await loadAllQuestions()
   const nameToId = Object.fromEntries(
-    questions.data.filter((q) => q.status == QuestionStatus.LIVE).map((q) => [q.title, q.pageid])
+    questions.data
+      .filter((q) => q.status == QuestionStatus.LIVE_ON_SITE)
```
Comment (mruwnik) [inline app/server-utils/stampy.ts:287]: I wonder if there should be a general filter function to handle this? I'm guessing there will be more such situations in the future :/

File: app/root.css [hunk 3]
```diff
@@ -181,7 +181,6 @@ button.icon-link,
 a:visited.icon-link {
   font-size: 0.6rem;
   text-transform: uppercase;
-  white-space: nowrap;
```
Comment (mruwnik) [inline app/root.css:184]: this is all black magic for me, but I assume it works

**Review by mruwnik — APPROVED** [review]

File: app/root.css (hunk 3 above)
Comment (Aprillion) [reply app/root.css:184]: it's half black magic to me too  🪄 

the use of `overflow-x: hidden` was triggering a change of default rule for `overflow-y: auto` causing `body` and `html` to be both scrollable on mobile screens 🙀 

...so I had to find a different alternative for overflow-x and I pinpointed this issue as causing the horizontal scrolling issue (a few px of overflow-x on <1000px window) by disabling 1 rule at a time for "a few minutes"

...and then I had to figure out another nowrap equivalent, so I changed the previous `flex: 0` (with implied flex-basis as "0%") to `flex: none` (with implied flex-basis as "auto" which defaults to the width of the child before wrapping - at least when some particular constellation of stars is most to our code)

🤞 hopefully no further chain reaction 🍀

File: app/hooks/useQuestionStateInUrl.ts (hunk 1 above)
Comment (Aprillion) [reply app/hooks/useQuestionStateInUrl.ts:126]: not really, but I no longer care today 🤷‍♀️

feel free to remove in next PR (or I will remove myself when I notice next time)

File: app/server-utils/stampy.ts (hunk 2 above)
Comment (Aprillion) [reply app/server-utils/stampy.ts:287]: sure, but it sounded like "a future problem" to me right now 😅

**Pull request merged**